UI Actions/Send Notification to assigned user/script.js

```diff
@@ -0,0 +1,9 @@
+//Server Side Script to send notification to the assigned to user
+
+var gr_sys_email = new GlideRecord('sys_email');
+gr_sys_email.initialize();
+gr_sys_email.type = 'send-ready';
+gr_sys_email.subject = 'UI Action Notification from Incident ' + current.number;
+gr_sys_email.recipients = current.assigned_to.email;
+gr_sys_email.body = 'As the incident ' + current.number + ' is assigned to you, this UI Action Notification has been sent. Please review the incident.';
```

Comment:
I'd suggest updating to use the setter and getter methods
```js
gr_sys_email.setValue('field_name', value);
current.getValue('field_name');
```

This avoids type casting